tests/integration/sentinel/transitions/registration.spec.js

```diff
@@ -98,7 +97,225 @@ describe('registration', () => {
+        phone_number: patient_phone
+      },
+      reported_date: moment().valueOf(),
+      contact: {
+        _id: 'person',
+        parent: { _id: 'clinic', parent: { _id: 'health_center', parent: { _id: 'district_hospital' } } }
+      }
+    };
+
+    const docs = [
+      patientNameAndInvalidPhone
+    ];
+    const docIds = getIds(docs);
+
+    return utils
+      .updateSettings(settings, 'sentinel')
+      .then(utils.saveDocs(docs))
+      .then(() => sentinelUtils.waitForSentinel(docIds))
+      .then(() => sentinelUtils.getInfoDocs(docIds))
+      .then(infos => {
+        infos.forEach(info => {
+          chai.expect(info).to.deep.nested.not.include({ 'transitions.registration.ok': false });
+        });
+      }).then(() => utils.getDocs(docIds))
+      .then(updated => {
+        chai.expect(updated[0].fields.phone_number).to.equal(patient_phone);
+        chai.expect(updated[0].patient_id).equal(undefined);
+      })
+      .then(patients => {
+        chai.expect(patients).to.be.undefined;
+      });
+  });
+
+  it('should fail transition on invalid phone', () => {
+    const patient_phone = '+9779666666666';
+    const settings = {
+      transitions: { registration: true },
+      registrations: [{
+        form: 'FORM-A',
+        events: [{
+          name: 'on_create',
+          trigger: 'add_phone_number',
+          params: 'phone_number',
+          bool_expr: 'doc.fields.phone_number'
+        }, {
+          name: 'on_create',
+          trigger: 'add_patient',
+          params: '',
+          bool_expr: ''
+        }],
+        messages: [{
+          recipient: 'reporting_unit',
+          event_type: 'report_accepted',
+          message: [{
+            locale: 'en',
+            content: 'Patient {{patient_name}} ({{patient_id}}) added to {{clinic.name}}'
+          }],
+        }],
+      }],
+      forms: { 'FORM-A': {} }
+    };
+
+    const patientNameAndInvalidPhone = { // has just the `patient_name` field, and should create this person
+      _id: uuid(),
+      type: 'data_record',
+      form: 'FORM-A',
+      from: '+9779841212345',
+      fields: {
+        patient_name: 'Minerva',
+        phone_number: patient_phone
+      },
+      reported_date: moment().valueOf(),
+      contact: {
+        _id: 'person',
+        parent: { _id: 'clinic', parent: { _id: 'health_center', parent: { _id: 'district_hospital' } } }
+      }
+    };
+
+    const docs = [
+      patientNameAndInvalidPhone
+    ];
+    const docIds = getIds(docs);
+
+    return utils
+      .updateSettings(settings, 'sentinel')
+      .then(utils.saveDocs(docs))
+      .then(() => sentinelUtils.waitForSentinel(docIds))
+      .then(() => sentinelUtils.getInfoDocs(docIds))
+      .then(infos => {
+        infos.forEach(info => {
+          chai.expect(info).to.deep.nested.not.include({ 'transitions.registration.ok': false });
+        });
+      });
+  });
+
+  it(' be skipped when transition is disabled', () => {
```

Comment:
Looks like you modified the test name accidentally?

```suggestion
  it('should be skipped when transition is disabled', () => {
```